vignettes/packageDiscrepencies.Rmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "Package discrepencies"
+author: "T. Joe Mills"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{Package discrepencies}
+  \usepackage[utf8]{inputenc}
+---
+
+This vignette summarizes discrepancies between the statistic calculations in the new (>v1.0) and the old (<v1.0) EflowStats packages. In general, the authors of the new EflowStats package focused primarily on ensuring that the statistics were calculated according to the definitions stated in Henriksen et al, 2006, rather than on reproducing output from previous package versions or the original HITHAT software. This process uncovered a number of issues with the old EflowStats statistics as well as some statistics calculated in the original HITHAT software (Henriksen et al, 2006). A record of this process can be found in the [Closed issues](https://github.com/USGS-R/EflowStats/issues?q=is%3Aissue+is%3Aclosed) page of the GitHub repository for this package.
```

Comment:
Reading through this, I wanted to remove some wordiness. PR is incoming in a bit.